shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -389,6 +418,30 @@ - (void)makeResourceContextCurrent {
   [_resourceContext makeCurrentContext];
 }
 
+- (BOOL)populateTextureWithIdentifier:(int64_t)textureID
```

Comment:
Nit: I'm trying to keep the definition orders in this file in logical groups, and consistent with their declaration order. So this method should be after dispatchKeyEvent:, as that's where it's declared (or the declaration above moved to match this, which would be fine as well), and the next three methods should go later in the file (I would put them after the `FLEPluginRegistrar` section) with a
```
#pragma mark - FlutterTextureRegistrar
```
above it, consistent with the other protocol implementations.